5/bin/entrypoint.sh

```diff
@@ -26,15 +27,14 @@ if [[ "$#" == 0 ]]; then
 fi
 
 # if first arg looks like a flag, assume we want to run sonar-scanner with flags
-if [[ "${1#-}" != "${1}" ]] || [[ -z "$(command -v "${1}")" ]]; then
+if [[ "${1#-}" != "${1}" ]] || ! command -v "${1}" > /dev/null; then
```

Comment:
I'm not sure about this change. I think that:

- the previous version `[[ -z "$(command -v "${1}")" ]]` is much more readable
- does it really work? I was testing it and it does not seem to work well...

Reply:
Can you provide a use case that does not work well?
Since we are using `set -o`, it is recommended to avoid this kind of construction: https://www.shellcheck.net/wiki/SC2312.

Reply:
We discussed this, let's use what shellcheck suggests :)